### PR TITLE
Fedora.28.Amd64 queue is dead. Use container instead

### PR DESCRIPTION
**Pull request by wfurt**
all Fedora test runs are failing now.

### CODE REVIEW

File: eng/platform-matrix.yml [hunk 1]
```diff
@@ -140,7 +140,7 @@ jobs:
           - Ubuntu.1604.Amd64
           - Ubuntu.1804.Amd64
           - Centos.7.Amd64
-          - Fedora.28.Amd64
+          - \(Fedora.28.Amd64\)ubuntu.1604.amd64@mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-28-helix-45b1fa2-20190327215722
```
Comment (echesakovMSFT) [inline eng/platform-matrix.yml]: Can we use the same case convention (Pascal.Case ?) for ubuntu.1604.amd64 (i.e. Ubuntu.1604.Amd64) to keep consistency with other places in the file?

**Comment by wfurt** [comment]
I'm not sure about the escaping. I noticed that https://mc.dot.net/#/product/netcore/30/source/official~2Fdotnet~2Fcoreclr~2Frefs~2Fheads~2Fmaster/type/test~2Ffunctional~2Fcli~2F/build/20190405.71 does not show the friendly name. 
I took this from corefx where it works. I can try change the casing but I don't know if Helix is case sensitive or not.

**Comment by echesakovMSFT** [comment]
> I'm not sure about the escaping. I noticed that https://mc.dot.net/#/product/netcore/30/source/official~2Fdotnet~2Fcoreclr~2Frefs~2Fheads~2Fmaster/type/test~2Ffunctional~2Fcli~2F/build/20190405.71 does not show the friendly name.

I think it could be due to the name in parenthesis being the same as the name of the "real" queue

`(Ubuntu.1604.Arm32)Ubuntu.1604.Arm32@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-16.04-helix-arm32v7-0a0ebdd-20190312215551-arm`

@MattGal What do you think?

> I took this from corefx where it works. I can try change the casing but I don't know if Helix is case sensitive or not.

Helix is not case-sensitive as far as I know - I was just trying to keep the queues names in the file consistent

**Comment by MattGal** [comment]
If you want to show the friendly name you need to use the "operatingSystem" property in the view config; it's possible CoreCLR is just showing the Queue ID.

**Comment by echesakovMSFT** [comment]
> If you want to show the friendly name you need to use the "operatingSystem" property in the view config; it's possible CoreCLR is just showing the Queue ID.

@MattGal Are you talking about this file https://github.com/dotnet/core-eng/blob/master/mission-control-config/dotnet/coreclr/viewconfiguration.json ?

**Comment by MattGal** [comment]
@echesakov I figured it out... the reason this isn't showing a pretty name is that you're using the same string as the pretty-printed version, so it's leaving your operatingSystem property alone.

I can change the API to make this a bit more aggressive pretty easily, but if you want results quickly simply change the contents of the parenthesis (i.e. append '.Docker" to the end or something)

**Comment by echesakovMSFT** [comment]
@MattGal Isn't this what I was suspecting ?

> I think it could be due to the name in parenthesis being the same as the name of the "real" queue
> (Ubuntu.1604.Arm32)Ubuntu.1604.Arm32@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-16.04-helix-arm32v7-0a0ebdd-20190312215551-arm

**Comment by MattGal** [comment]
Yes, and you did a great job suspecting the correct thing;  earlier incarnations of the feature didn't allow specifying a queue that actually existed, so this was overlooked.

**Pull request merged**